supporting_files/5gmag-rt.conf

```diff
@@ -42,7 +42,7 @@ rp: {
   }
 }
 
-gw: {
+mw: {
   cache: { 
     base_path: "/tmp/obeca";
```

Comment:
Suggest to change this path to `tmp/5gmag-rt`

Reply:
base_path is obsolete, files are only cached in-memory now. Removing.